src/ports/nfts/rentalsNFTComponent.ts

```diff
@@ -0,0 +1,140 @@
+import { Contract, NFTFilters } from '@dcl/schemas'
+import { IRentalsComponent, RentalAsset } from '../rentals/types'
+import { INFTsComponent, NFTResult } from './types'
+
+// The recommended maximum amount of elements a subgraph filter_in can handle.
+// For example, if we had a tokenId_in filter, the maximum amount of tokenIds provided would be this number.
+export const MAX_SUBGRAPH_QUERY_IN_ELEMENTS = 500
+
+/**
+ * This component allows users to retrieve Lands and Estates by owner that are currently owned by the rentals contract.
+ * Once a rental starts, the asset is transferred to the rentals contract, so when we want to use the nft server to retrieve
+ * the assets owned by an address, conventionally, the assets that were rented would not be returned because the owner is the rentals
+ * contract.
+ * This component will first fetch the rentals subgraph to check which assets are currently rented by a given address, and then look them up
+ * in the marketplace subgraph to retrieve all the nft data.
+ * @param options.rentalsComponent - The rentals component which allows querying the rentals subgraph.
+ * @param options.marketplaceNFTsComponent - The marketplace component which allows querying the marketplace subgraph.
+ * @param options.contractAddresses - The addresses of the Land and Estate contracts which will be used to filter the results.
+ */
+export function createRentalsNFTComponent(options: {
```

Comment:
The one for marketplace NFTs is called `createNFTComponent`, I think the component part should be there.

It is called Rental instead of Rented on the one hand to match the RentalAsset entity of the subgraph and because being in the RentalsContract does not mean it is Rented, it means that it is an asset used for Rentals

Reply:
Even if it was there, I think that we need to split it into another component (`rental-nfts`), as all the other components follow the same directory structure:
- component.ts
- types.ts
- utils.ts

You're right, it doesn't really have to be `rented` as the nfts could not be rented and be on the rentals contract.